src/components/BibPage/BibDetail.tsx

```diff
@@ -13,14 +21,53 @@ const buildDetailElement = (field: BibDetail) => {
   return (
     <>
       <dt>{field.label}</dt>
-      {field.value.map((val: string, i: number) => {
-        const stringDirection = displayRtl(val)
-        return (
-          <dd dir={stringDirection} key={i}>
-            {val}
-          </dd>
-        )
-      })}
+      <dd>
+        <List data-testId={`${field.label}`} type="ol" noStyling>
+          {field.value.map((val: string, i: number) => {
+            const stringDirection = displayRtl(val)
+            return (
+              <li dir={stringDirection} key={i}>
+                {val}
+              </li>
+            )
+          })}
+        </List>
+      </dd>
+    </>
+  )
+}
+
+const buildSingleSubjectHeadingElement = (subjectHeadingUrls: Url[]) => {
+  const urls = subjectHeadingUrls.reduce((linksPerSubject, url: Url, index) => {
+    // Push a divider in between the link elements
+    const divider = (
+      <span data-testid="divider" key={`divider-${index}`}>
+        {" "}
+        &gt;{" "}
+      </span>
+    )
+    const link = linkElement(url, "internal")
+    linksPerSubject.push(link)
+    if (index < subjectHeadingUrls.length - 1) linksPerSubject.push(divider)
+    return linksPerSubject
+  }, [] as React.JSX.Element[])
+  return urls
+}
+
+const buildCompoundSubjectHeadingElement = (field: SubjectHeadingDetail) => {
+  const subjectHeadingLinksPerSubject = field.value.map(
+    (subjectHeadingUrls: Url[]) => {
+      return buildSingleSubjectHeadingElement(subjectHeadingUrls)
+    }
+  )
+  return (
+    <>
+      <dt>{field.label}</dt>
+      {subjectHeadingLinksPerSubject.map((subject, i) => (
+        <Box data-testid="subjectLinksPer" key={"subject-" + i}>
+          {subject}
+        </Box>
```

Comment:
Related but in the `buildLinkedElement` function:
wrap `return linkElement(urlInfo, field.link)` in a `dd` so the definition aligns.
<img width="317" alt="Screen Shot 2024-01-04 at 5 09 39 PM" src="https://github.com/NYPL/research-catalog/assets/1280564/bebc8bf5-fe54-4934-9ef9-ed4ed5976d93">

Reply:
i was wondering why that indentation was different... thanks!